pkg/controller/controller.go

```diff
@@ -701,7 +701,10 @@ func (p *csiProvisioner) prepareProvision(ctx context.Context, claim *v1.Persist
 			p.csiNodeLister,
 			p.nodeLister,
 			p.pvcNodeStore)
-		if err != nil {
+		if apierrors.IsNotFound(err) {
+			// The node or CSINode object can't be found, ask the scheduler for a reschedule
+			return nil, controller.ProvisioningReschedule, err
```

Comment:
Does the GenerateAccessibilityRequirements properly return NotFound error now if the CSINode object does not exist in the cache?

Reply:
Under what condition the node never comes back will retry forever? Because if it is a long provisioning, the PV object will be eventually created or failed with a final error, so it should not be forever?